src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        return new Circle("WHITE", 10);
+    }
+
+    public int getNumber() {
+        return random.nextInt(10);
+    }
+
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        String color = colorSupplier.getRandomColor();
+        int figureNumber = random.nextInt(FigureName.values().length);
```

Comment:
we can use FigureName to generate `figureNumber` or we can create and use constant to do the same
```suggestion
        int figureNumber = random.nextInt(NUMBER_OF_FIGURES);
```